### ISSUE TITLE
Getting `[Errno 86] Bad CPU type in executable` on updating to v0.11.0

### ISSUE THREAD

**Issue by prayashm** [tilmanginzel/alfred-bluetooth-workflow#27]
Debug logs:
```
[21:14:11.697] Logging Started...
[21:14:14.037] Bluetooth Connector[Script Filter] Queuing argument ''
[21:14:14.277] Bluetooth Connector[Script Filter] Script with argv '(null)' finished
[21:14:14.285] ERROR: Bluetooth Connector[Script Filter] Code 1: .
21:14:14 workflow.py:2055 DEBUG    ---------- Bluetooth Connector (0.11.0) ----------
21:14:14 workflow.py:1462 DEBUG    reading settings from /Users/prayash/Library/Application Support/Alfred/Workflow Data/de.tilmanginzel.alfred.bluetooth/settings.json
21:14:14 workflow.py:2339 DEBUG    update check not due
21:14:14 workflow.py:1689 DEBUG    loading cached data: /Users/prayash/Library/Caches/com.runningwithcrayons.Alfred/Workflow Data/de.tilmanginzel.alfred.bluetooth/__workflow_update_status.cpickle
21:14:14 workflow.py:2269 DEBUG    update_data: {u'available': False, u'version': u'v0.11.0', u'download_url': u'https://github.com/tilmanginzel/alfred-bluetooth-workflow/releases/download/v0.11.0/Bluetooth.Connector.alfredworkflow'}
21:14:14 workflow.py:2074 ERROR    [Errno 86] Bad CPU type in executable
Traceback (most recent call last):
  File "/Users/prayash/Library/Application Support/Alfred/Alfred.alfredpreferences/workflows/user.workflow.4BFFCC90-360E-4723-A3F9-B6067A813D63/workflow/workflow.py", line 2067, in run
    func(self)
  File "alfred_bluetooth_workflow.py", line 40, in main
    devices = _read_devices()
  File "alfred_bluetooth_workflow.py", line 12, in _read_devices
    proc = subprocess.Popen(['./blueutil', '--paired', '--format=JSON'], stdout=subprocess.PIPE)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/subprocess.py", line 394, in __init__
    errread, errwrite)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/subprocess.py", line 1047, in _execute_child
    raise child_exception
OSError: [Errno 86] Bad CPU type in executable
21:14:14 workflow.py:2097 DEBUG    ---------- finished in 0.036s ----------
[21:14:14.289] Bluetooth Connector[Script Filter] {"items": [{"valid": false, "subtitle": "[Errno 86] Bad CPU type in executable", "icon": {"path": "/System/Library/CoreServices/CoreTypes.bundle/Contents/Resources/AlertStopIcon.icns"}, "title": "Error in workflow 'Bluetooth Connector'"}]}
```

**Comment by thatguychrisw** [comment]
The issue seems to be related to the `blueutil` binary update in version v0.11.0.  I opened the workflow's folder and replaced that file with the one from [version v0.10.0](https://github.com/tilmanginzel/alfred-bluetooth-workflow/releases/tag/v0.10.0) and it's working again.

**Comment by tilmanginzel** [comment]
Hi @prayashm, @thatguychrisw, thanks for making me aware of it. I have compiled the binary on a M1 macbook, that's probably the root cause. I will publish a new version latest tomorrow.

**Comment by cameronsstone** [comment]
Thanks for jumping on this so quickly. :-)

**Comment by tilmanginzel** [comment]
I have just published version `0.11.1` with a new binary, tested it on a M1 and Intel Macbook. Should be fixed now :)
